src/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Unix.cs

```diff
@@ -74,7 +74,7 @@ internal static void ThrowIfRemoteMachine(string machineName)
         {
             if (IsRemoteMachine(machineName))
             {
-                throw new PlatformNotSupportedException(SR.RemoteMachinesNotSupported);
+                throw new InvalidOperationException(SR.CouldntConnectToRemoteMachine);
```

Comment:
Do we really want to change this one?  It's an unsupported feature on Unix; throwing saying we couldn't connect suggests we tried.